awesome_owl/static/src/playground.js

```diff
@@ -1,7 +1,19 @@
 /** @odoo-module **/
 
-import { Component } from "@odoo/owl";
+import { Component, markup, useState } from "@odoo/owl";
+import { Counter } from './Counter/counter';
+import { Card } from './Card/card';
+import { TodoList } from "./ToDo/todo_list";
+
 
 export class Playground extends Component {
     static template = "awesome_owl.playground";
+    static components = { TodoList, Card, Counter };
+
+    html = markup("<div>some content</div>")
+    counters_sum = useState({ value: 0 });
```

Comment:
nitpick: js-side use camelCase for variables
```suggestion
    countersSum = useState({ value: 0 });
```